develop/tutorials/articles/300-osgi-basics/02-using-the-wab-generator.markdown

```diff
@@ -17,11 +17,25 @@ OSGi directive. Although the source of the project has a WAR layout, the
 artifact filename may end with either the `.jar` or `.war` extension.
 
 Liferay only supports the use of WABs if they've been auto-generated by the WAB
-Generator. The WAB Generator transforms a general WAR-style portlet into a WAB
+Generator. The WAB Generator transforms a general WAR-style plugin into a WAB
 during its deployment process. So what exactly does the WAB Generator do to a
 WAR file to transform it into a WAB?
 
-Consider the following folder structure of a WAR-style portlet:
+First, the WAB Generator identifies and specifies the plugin's dependencies. It
+analyzes the plugin's JSPs and descriptor files to identify the packages the
+plugin requires. The descriptor files include `web.xml`, `liferay-web.xml`,
+`portlet.xml`, `liferay-portlet.xml`, and `liferay-hook.xml`.  The WAB Generator
+adds each package to an `Import-Package` header in the WAB's
+`META-INF/MANIFEST.MF` file.
+
+Next, the WAB Generator finds and registers the plugin's extension point
+information. It registers portlet extension points declared in `portlet.xml` and
+`liferay-portlet.xml` files. It registers extension point implementations
+declared in Hook `liferay-hook.xml` files. 
```

Comment:
I would remove this paragraph since all this is handled elsewhere, not the WAB generator.